src/mono/wasm/host/Program.cs

```diff
@@ -30,6 +30,9 @@ public static async Task<int> Main(string[] args)
         RegisterHostHandler(WasmHost.Wasmtime, WasiEngineHost.InvokeAsync);
 
         using CancellationTokenSource cts = new();
+
+        Console.CancelKeyPress += (object? sender, ConsoleCancelEventArgs e) => cts.Cancel();
```

Comment:
@radical Without this line, current WasmAppHost won't terminate on Ctrl+C pressed in console, when there is a console forward + client is connected / ws is opened